server/views/temporary-accommodation/assessments/show.njk

```diff
@@ -8,14 +9,50 @@
 {% block content %}
   {% include "../../_messages.njk" %}
 
-  {{
-    mojIdentityBar({
-      title: {
-        html: "<span><h1>" + assessment.application.person.name + "</h1>" + statusBadge + "</span>"
-      },
-      menus: [{items: actions}]
-    })
-  }}
+  <div class="moj-identity-bar">
+    <div class="moj-identity-bar__container">
+      <div class="moj-identity-bar__details">
+        <span class="moj-identity-bar__title">
+          <span class="govuk-caption-xl">Review referral</span>
+          <h1 class="govuk-heading-xl">{{ assessment.application.person.name }}
+            {{ statusBadge | safe }}</h1>
```

Comment:
I'd like to come back to this and better understand why this can't be inserted using the Nunjucks component